### DISCUSSION TITLE
How to cancel the default event of key N?

### DISCUSSION THREAD

**Discussion by BoYs-J** [squidfunk/mkdocs-material#4949] [Q&A]
My web page needs to listen to keystroke N but it will be triggered at the same time as going to the previous page, which does not have the desired effect. Is there any way to unbind the button to go to the previous page?Because I am using GitHub Pages, the build page uses the pip installed release package. I modified bundle.ts, but it didn't work.
![work](https://user-images.githubusercontent.com/75466116/215409510-ca03ce39-0d88-4f65-b1f5-78e65c84f5eb.jpg)

**Comment by squidfunk** [comment]
It's currently not possible to override the default keys without theme customization. You would need to go through the [theme development process](https://squidfunk.github.io/mkdocs-material/customization/#theme-development) and [rebuild the theme](https://squidfunk.github.io/mkdocs-material/customization/#building-the-theme).
